prow/tide/tide.go

```diff
@@ -173,6 +241,17 @@ func (c *Controller) syncSubpool(ctx context.Context, sp subpool) error {
 		}
 		presubmits = append(presubmits, ps.Name)
 	}
+	batchMerge, batchPending := accumulateBatch(presubmits, sp.prs, sp.pjs)
+	if batchPending {
+		c.log.Info("Waiting for batch to complete.")
```

Comment:
I guess it's intentional to not proceed with doing anything else for this branch if a batch job is pending. Add a comment to that extend.

Reply:
Done.